test/mock-agent.js

```diff
@@ -910,6 +1237,30 @@ test('MockAgent - close removes all registered mock clients', async (t) => {
   }
 })
 
+test('MockAgent - close removes all registered mock call history', async (t) => {
+  t = tspl(t, { plan: 6 })
+
+  const mockAgent = new MockAgent()
+  setGlobalDispatcher(mockAgent)
+
+  const mockClient = mockAgent.get('http://localhost:9999')
+
+  mockClient.intercept({
+    path: '/foo',
+    method: 'GET'
+  }).reply(200, 'foo').registerCallHistory('my-history')
+
+  t.strictEqual(MockCallHistory.AllMockCallHistory.size, 2)
+  t.ok(mockAgent.getCallHistory() instanceof MockCallHistory)
+  t.ok(mockAgent.getCallHistory('my-history') instanceof MockCallHistory)
+
+  await mockAgent.close()
+
+  t.strictEqual(MockCallHistory.AllMockCallHistory.size, 0)
+  t.strictEqual(mockAgent.getCallHistory(), undefined)
```

Comment:
type here is `MockCallHistory` with using `getCallHistory` without `name` parameter. Can you confirm me that a closed agent could not be used anymore ?

Reply:
A closed Agent can be re-used

Reply:
Ok i changed that too so